src/test/java/com/myhome/controllers/unit/HealthCheckControllerTest.java

```diff
@@ -0,0 +1,25 @@
+package com.myhome.controllers.unit;
+
+import com.myhome.controllers.HealthCheckController;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.MockitoAnnotations;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class HealthCheckControllerTest {
+
+  @InjectMocks
+  HealthCheckController healthCheckController;
+
+  @BeforeEach
+  void init() {
+    MockitoAnnotations.initMocks(this);
+  }
+
+  @Test
+  void shouldReturnHealthySuccessfully() {
+    assertEquals("Application is healthy", healthCheckController.statusCheck());
```

Comment:
Would be great to change implementation of health check to return the `ResponseEntity<String>`.

Also the message should be extracted to variable and `.statusCheck()` request.
Please apply given when then pattern

Reply:
@mslowiak I don't see where the given pattern should come in here since statusCheck() always returns Application is healthy so there is no given(Mock method).willReturn required. Or should there be some HealthCheckService? But I think that may be a bit overkill

Reply:
Yes, I was thinking about solution that you provided in code snippet